app/controllers/phases_controller.rb

```diff
@@ -176,7 +179,8 @@ def admin_update
     if @phase.update_attributes(params[:phase])
       redirect_to admin_show_phase_path(@phase), notice: _('Information was successfully updated.')
     else
-      render action: "admin_show"
+      # Redirecting here because the method loads a lot of stuff. We may lose whatever the user had entered though 
```

Comment:
that's a good point. The redirect would likely lose whatever the user had entered. I'll switch them to renders